### DISCUSSION TITLE
Use npm

### DISCUSSION THREAD

**Discussion by bridevmx** [benallfree/pocketpages#7] [Q&A]
Hello, has anyone used "https://github.com/benallfree/pocketpages/"  I see that it uses bun, and I'm somewhat of a novice, so I don't understand how to use it with npm or how to start a new project. Could someone please help me?

**Comment by benallfree** [comment]
Hello, npm should work the same way but yes maybe it would be helpful to show examples for all package managers. Thoughts?

**Reply by bridevmx** [reply]
Thank you for responding. I forgot to congratulate you on your project. I find it so fitting that we understand that sometimes going back to basics and essentials is not a bad thing. As more functions and new technologies come out day by day, they can be overwhelming (at least for me, as I'm a novice and self-taught programmer). This project returns to the essence while being functional and offering a rapid development approach.

I have a problem in that I don't fully understand the steps for installing with "npm". I've tried substituting "bunx", but it doesn't work for me. Hopefully, in the future, you can add this to your documentation. Again, congratulations and thank you for your contribution.

**Reply by benallfree** [reply]
Thank you that is very kind. I'll be sure to update this thread when I'm ready for you to try again.

**Comment by benallfree** [comment]
v0.12 has been released and should be much easier to use :)
